test/api/utils/check-security-headers.js

```diff
@@ -23,8 +23,7 @@ export function checkSecurityHeaders(request: Test) {
     .expect('X-DNS-Prefetch-Control', 'off')
     .expect('X-Frame-Options', 'DENY')
     .expect('X-Download-Options', 'noopen')
-    .expect('X-Content-Type-Options', 'nosniff')
-    .expect('X-XSS-Protection', '1; mode=block');
```

Comment:
This was removed because didn't prevent much. I did my reseach and found this issue discussing this change: https://github.com/helmetjs/helmet/issues/230, so I decided to not add it back.